src/FileUpload/FileNameGenerator/Slug.php

```diff
@@ -90,28 +94,359 @@ public function getSluggedFileName($name)
         return $this->slugify($fileNameExploded) . "." . $extension;
     }
 
+
     /**
-     * @param $text
+     * Create a web friendly URL slug from a string.
      *
-     * @return mixed|string
+     * Although supported, transliteration is discouraged because
+     *     1) most web browsers support UTF-8 characters in URLs
+     *     2) transliteration causes a loss of information
+     *
+     * @author Sean Murphy <sean@iamseanmurphy.com>
+     * @copyright Copyright 2012 Sean Murphy. All rights reserved.
+     * @license http://creativecommons.org/publicdomain/zero/1.0/
+     *
+     * @param string $str
+     * @param array $options
+     *
+     * @return string
      */
-    private function slugify($text)
+    function slugify($str, $options = array())
```

Comment:
i meant changing `function slug()` to `private function slug()` .. I don't seem to find the change